tests/py/test_www_team_distributing.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from gratipay.testing import Harness
+
+
+class Tests(Harness):
+
+    def test_distributing_redirects_when_no_money_is_available(self):
+        self.make_team()
+        assert self.client.GxT('/TheEnterprise/distributing/').code == 302
+
+    def test_distributing_doesnt_redirect_when_money_is_available(self):
+        self.make_team()
+        self.db.run("UPDATE teams SET available=537")
+        assert self.client.GET('/TheEnterprise/distributing/', raise_immediately=False).code == 500
```

Comment:
> which is [still](https://github.com/gratipay/gratipay.com/blob/1976/tests/py/test_www_team_distributing.py#L32-L35) on `master`

(It's fixed in 77e8334c72e7dfeea03687e779971d3f39b26133 on #4077, which is still a work in progress.)